CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Change Log
 
+## 7.17.0 (October 29, 2017)
+
+- Fix: Fixed issue with `vscode.nls`. ([@JimiC](https://github.com/JimiC) in [#1319](https://github.com/vscode-icons/vscode-icons/pull/1319))
+- Enhancement: Updated dependencies. ([@JimiC](https://github.com/JimiC) in [#1317](https://github.com/vscode-icons/vscode-icons/pull/1317))
+- Feature: Support for `Apex`. ([@JimiC](https://github.com/JimiC) in [#1316](https://github.com/vscode-icons/vscode-icons/pull/1316))
+- Feature: Support for `Mediawiki`. ([@JimiC](https://github.com/JimiC) in [#1315](https://github.com/vscode-icons/vscode-icons/pull/1315))
+- Feature: Support for `Mojolicious`. ([@JimiC](https://github.com/JimiC) in [#1314](https://github.com/vscode-icons/vscode-icons/pull/1314))
+- Fix: PAD disabled detectigon flow. ([@robertohuertasm](https://github.com/robertohuertasm) in [#1312](https://github.com/vscode-icons/vscode-icons/pull/1312))
```

Comment:
Typo on `detection`,

Reply:
Also, use more descriptive description like `Fixed PAD disabled detection flow.`

Reply:
Any response to the above comment? Did you miss it?

Reply:
Yep, I totally missed it.